ci/format.bat

```diff
@@ -28,5 +28,4 @@ cd "%ci_dir%"
 REM Do not use the CALL command in the next line to execute Dart. CALL causes
 REM Windows to re-read the line from disk after the CALL command has finished
 REM regardless of the ampersand chain.
-REM TODO(gspencergoog): Remove --no-sound-null-safety once isolate package is null-safe.
-"%dart%" --no-sound-null-safety --disable-dart-dev bin\format.dart %* & exit /B !ERRORLEVEL!
+"%dart%" --disable-dart-dev bin\format.dart %* & exit /B !ERRORLEVEL!
```

Comment:
It may be worth moving the null safety switch to a separate PR so that if we have to revert one we don't need to revert the other.

Reply:
It seems that this was changed in main separately from this PR already.